posts/2019-05-15-4-Years-Of-Rust.md

```diff
@@ -0,0 +1,42 @@
+
+Releasing Rust with stability guarantees also meant putting a stop to large visible changes. The face of Rust is still very similar to Rust 1.0. Even with the last year’s 2018 Edition idioms, Rust is still very recognizable as what it was in 2015. That steadiness hides that the time of Rust’s fastest development and growth is *now*. With the stability of the language and easy upgrades as a base, a ton of new features have been built. We’ve seen a bunch of achievements in the last year:
+
+
+- We have been StackOverflow’s [“Most loved programming language”][stackoverflow] 4 consecutive years in a row
+- We opened up a whole new area of development for stable Rust: [embedded development][rust-embedded]
+- [Rust+WASM][rust-wasm] went from an experiment to a usable product, making rustc the first compiler with focus on supporting WASM
+- We shipped a whole new language edition: [Rust 2018][rust-2018]
+- [Crates.io][crates-io] passed a billion downloads and has over 25,000 crates available
+- There’s now over 100 meetups around the world
+- 6(!) new conferences were spun up ([RustRush][rustrush], [RustCon Asia][rustcon-asia], [Oxidize][oxidize], [Rust LATAM][rust-latam], [Colorado Gold Rust][coloradogoldrust], [RustLab Italy][rustlab])
+
+This list could go on and on. It would be presumptuous to attribute all of these successes to the Rust project. Indeed, we are slowly entering a phase where much of the new and innovative development happens outside of the compiler itself. This includes projects like whole game engines, but also many small, helpful libraries. Birthdays are a great time to take a look back over the last year and see the happy parts!
+
+Rust would be nothing, and especially not winning prizes, without its community. Community happens everywhere! We would like to thank everyone for being along on this ride, from project members to small scale contributors to people just checking the language out and finding interest in it. Your interest and curiosity is what makes the Rust community an enjoyable place to be. Some meetups [are running birthday parties][calendar] today to which everyone is invited. If you are not attending one, you can take the chance to celebrate in any other fashion: maybe show us a picture of what you are currently working on or talk about what excites you. If you want to take it to social media, consider tagging our [Twitter account][twitter] or using the hashtag #rustbirthday.
```

Comment:
I'm not sure what a "project member" is - it's a definition I'm a bit uncomfortable with because I think the smallest scale contributor is still a contributor

Reply:
team member?

Reply:
```suggestion
Rust would be nothing, and especially not winning prizes, without its community. Community happens everywhere! We would like to thank everyone for being along on this ride, from team members to small scale contributors to people just checking the language out and finding interest in it. Your interest and curiosity is what makes the Rust community an enjoyable place to be. Some meetups [are running birthday parties][calendar] today to which everyone is invited. If you are not attending one, you can take the chance to celebrate in any other fashion: maybe show us a picture of what you are currently working on or talk about what excites you. If you want to take it to social media, consider tagging our [Twitter account][twitter] or using the hashtag #rustbirthday.
```